src/djcytoscape/tests/test_fields.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.contenttypes.models import ContentType
+
+from django_tenants.test.cases import TenantTestCase
+
+from djcytoscape.models import CytoScape
+
+
+class AllowedContentObjectChoiceFieldTest(TenantTestCase):
+
+    def test_queryset(self):
```

Comment:
docstring needed

Reply:
why do yoo need docstring in tests?

Reply:
It's helpful for other developers, it prints when a test fails, providing useful info and saving time, also see:
https://hynek.me/articles/document-your-tests/
https://jml.io/test-docstrings/

Reply:
However, I realize this convention is not documented in the current contributing.md, so if you want to skip it, you can.